### PR TITLE
Homework3

### PR DESCRIPTION
**Pull request by pprzybys2**
Homework3: pointers, slice, map.

### CODE REVIEW

File: module-3/task1/go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module task1
+
+go 1.14
```
Comment (wshaman) [inline module-3/task1/go.mod:3]: why it's 14 and not 16?

File: module-3/task3/maps/maps.go [hunk 2]
```diff
@@ -1,4 +1,20 @@
...
 
+import (
+	"sort"
+	"strings"
+)
+
 func SortedValues(m map[int]string) string {
+	values := make([]string, 0, len(m))
+	keys := make([]int, 0, len(m))
+	for i, _ := range m {
```
Comment (wshaman) [inline module-3/task3/maps/maps.go:11]: second `_` can be omitted.

File: module-3/task3/maps/maps.go [hunk 3]
```diff
@@ -1,4 +1,20 @@
...
+		keys = append(keys, i)
+	}
+	sort.Ints(keys)
+	for _, k := range keys {
+		values = append(values, m[k])
+	}
+	concatenate := strings.Join(values[:], "")
```
Comment (wshaman) [inline module-3/task3/maps/maps.go:18]: can we use `values` instead of `values[:]` ?

**Review by wshaman — COMMENTED** [review]
:+1:

File: module-3/task1/go.mod (hunk 1 above)
Comment (pprzybys2) [reply module-3/task1/go.mod:3]: I have go version 1.14 installed. I think I'll update it.

**Pull request merged**